integration_tests/test_base_url.py

```diff
@@ -1,6 +1,19 @@
 import os
-
+import socket
 import requests
+import platform
+
+
+def get_network_host():
+    hostname = socket.gethostname()
+    ip_address = socket.gethostbyname(hostname)
+    # windows return ip address else return 0.0.0.0
+    # windows doesn't support 0.0.0.0
+
+    if platform.system() == "Windows":
+        return ip_address
+    else:
+        return "0.0.0.0"
```

Comment:
You should probably import the method from `conftest.py` instead of redefining it here. Or maybe move the method to another file and use it both here and in `conftest.py` if you want to keep only the fixtures and methods used only by fixtures in the `conftest.py` file.